cpp/src/parquet/properties.h

```diff
@@ -938,13 +937,13 @@ class PARQUET_EXPORT ArrowWriterProperties {
     /// list types (default "item"), will use "entries", as is specified in
```

Comment:
It seems this docstring is incorrect as per the Parquet spec, or am I reading it wrong?
```suggestion
    /// list types (default "item"), will use "element", as is specified in
```

See https://github.com/apache/parquet-format/blob/master/LogicalTypes.md#lists

Reply:
Oops.

```c++
MapType::MapType(std::shared_ptr<DataType> key_type, std::shared_ptr<Field> item_field,
                 bool keys_sorted)
    : MapType(::arrow::field("key", std::move(key_type), false), std::move(item_field),
              keys_sorted) {}

MapType::MapType(std::shared_ptr<Field> key_field, std::shared_ptr<Field> item_field,
                 bool keys_sorted)
    : MapType(
          ::arrow::field("entries",
                         struct_({std::move(key_field), std::move(item_field)}), false),
          keys_sorted) {}
```

Found `"entries"` in `arrow/type.cc`, and spec is using "element". 

And here maybe we're using "entries":

```c++
/// In a field with Map type, the field has a child Struct field, which then
/// has two children: key type and the second the value type. The names of the
/// child fields may be respectively "entries", "key", and "value", but this is
/// not enforced.
///
/// Map
/// ```text
///   - child[0] entries: Struct
///     - child[0] key: K
///     - child[1] value: V
/// ```
/// Neither the "entries" field nor the "key" field may be nullable.
```

According to spec. Map should use "key_value". Seems it's incorrect? Am I right?

Reply:
You are right, that should be "element".

Parquet will always have them as "element". Arrow has it's own spec, but isn't prescriptive for the name of the list type. Either way, I don't think we should care too much about this detail; these field names should be pretty much invisible to users.